browser/brave_local_state_prefs.cc

```diff
@@ -74,7 +75,14 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
 
 #if BUILDFLAG(BRAVE_P3A_ENABLED)
   brave::BraveP3AService::RegisterPrefs(registry,
+#if !defined(OS_ANDROID)
                                         first_run::IsChromeFirstRun());
+#else
+                                        // BraveP3AService::RegisterPrefs
+                                        // doesn't use this arg on Android
```

Comment:
I guess we will have to use something like this?

Reply:
I didn't find `first_run::IsChromeFirstRun()` version working on Android, so I have choosen another way.

The purpose of `bool first_run` argument is to be the default value for registrating `kP3ANoticeAcknowledged` pref.

So if it is `false` then informer will not be shown 
https://github.com/brave/brave-core/blob/master/browser/infobars/brave_confirm_p3a_infobar_delegate.cc#L31

in favor to get the settings from FirstTimeRunWizard


On Android `kP3ANoticeAcknowledged` is defaulted to false. 

In this PR similar is solved at the place
https://github.com/brave/brave-core/pull/7016/files#diff-448df90d6aa5a20513ca5dba42a4e428dfd22f05577b904f8fe7986c40ab90faR273

by keeping disabled code for future when onboarding UI for P3A will be implemented 
```
if (!PackageUtils.isFirstInstall(this)) {
     BraveP3AInformers.show();
}
```